bindings/wasm/test/account.spec.ts

```diff
@@ -35,12 +35,10 @@ async function run() {
 
     expect(account.getMetadata().alias).toBe('Alice');
 
-    account.setAlias('new alias');
-    expect(account.getMetadata().alias).toBe('new alias');
-
     const balance: AccountBalance = await account.sync();
-    expect(balance.baseCoin.available).toBe('0');
+    expect(balance.baseCoin.available).not.toBeNaN();
```

Comment:
Why doesn't this check for 0 anymore?

Reply:
because we dont care about hte balance, in a private tangle it could have. as long as it returns a number im cool with it 👍